internal/plugin/service/kafka/models.go

```diff
@@ -0,0 +1,38 @@
+package kafka
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type kafkaModel struct {
+	Timeouts              timeouts.Value `tfsdk:"timeouts"`
+	ID                    types.String   `tfsdk:"id"`
+	AdditionalDiskSpace   types.String   `tfsdk:"additional_disk_space"`
+	CloudName             types.String   `tfsdk:"cloud_name"`
+	Components            types.Set      `tfsdk:"components"`
+	DefaultACL            types.Bool     `tfsdk:"default_acl"`
+	DiskSpaceCap          types.String   `tfsdk:"disk_space_cap"`
```

Comment:
consider changing these to `types.Int64` as well if they represent numeric values